packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -604,8 +609,38 @@ class RawKeyboard {
     _listeners.remove(listener);
   }
 
-  /// A handler for hardware keyboard events that will stop propagation if the
-  /// handler returns true.
+  final List<RawKeyEventHandler> _keyHandlers = <RawKeyEventHandler>[];
+
+  /// Register a handler that is called when the user presses or releases
+  /// a hardware keyboard key if the [keyEventHandler] didn't handle it.
+  ///
+  /// These events will be delivered after the [keyEventHandler] (typically set
+  /// by the [FocusManager]) has returned false. If the [keyEventHandler]
+  /// returns true, then handlers registered in this way will not be called, as
+  /// the key is considered handled already.
+  ///
+  /// If any registered handler returns true, then the key event will be
+  /// considered handled by the platform, but all registered handlers will still
+  /// be called.
+  ///
+  /// If you want to listen to key events, but not handle them, use
+  /// [addListener] instead.
+  ///
+  /// Key handlers can be removed with [removeKeyHandler].
+  void addKeyHandler(RawKeyEventHandler handler) {
+    _keyHandlers.add(handler);
+  }
+
+  /// Stop calling the given handler every time the user presses or releases a
```

Comment:
nit: the handler wasn't called every time a key is pressed, right? Only if `keyEventHandler` didn't handle it, etc.